ai-models/tests/test_risk_level.py

```
@@ -0,0 +1,33 @@
+import pytest
+
+import train_model
+
+
+@pytest.fixture(scope="module")
+def predictor():
+    return train_model.WaterQualityPredictor.__new__(train_model.WaterQualityPredictor)
+
+
+@pytest.mark.parametrize(
+    "parameter,value,expected",
```

Ruff (0.15.21)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)